src/config/index.ts

```diff
@@ -54,6 +54,7 @@ const config = (): Configuration => {
 		cookieMaxAge: parseInt(getEnvValue('COOKIE_MAX_AGE', true), 10),
 		redisConnectionString: getEnvValue('REDIS_CONNECTION_STRING', false),
 		elasticSearchUrl: getEnvValue('ELASTICSEARCH_URL', true),
+		ssumRegistrationPage: getEnvValue('SSUM_REGISTRATION_PAGE', true),
```

Comment:
Add ENV var for all ENVs (INT, TST, QAS, PRD) on ibm cloud

Reply:
done